### PR TITLE
Add dependency and installation instructions

### PR DESCRIPTION
**Pull request by sleipnir**

### CODE REVIEW

**Review by marcellanz — COMMENTED** [review]
Thanks @sleipnir that looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,27 @@
...
+Cloning into 'python-support'...
+
+> cd python-support
+> python3 -m venv ./venv 
+> source ./venv/bin/activate
+> python --version     
+Python 3.7.3
+> pip --version 
+> pip install -r requirements.txt
+```
```
Comment (marcellanz) [inline README.md:18]: I cloned a repo and after this step, I could not start `./tck.sh` but after `pip install .` I where.

Comment (marcellanz) [reply README.md:18]: @sleipnir wdyt?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,27 @@
...
+
+### generate installer
+```
+./venv/bin/python3 setup.py bdist_wheel
```
Comment (marcellanz) [inline README.md:22]: @sleipnir 
I get this executing bdist_wheel. I read I need ti `pip install wheel`?
```
(venv) ➜  python-support git:(master) ✗ ./venv/bin/python3 setup.py bdist_wheel
package name: cloudstate, version: 0.5.0
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: invalid command 'bdist_wheel'
```

Comment (marcellanz) [reply README.md:22]: Yeah, after
```
pip install wheel
```

the next command installed the module.

**Comment by marcellanz** [comment]
@sleipnir please find two comments above, after I tried it locally. I have nearly nothing python wise on my box, and this might be the reason the `wheel` module is not available on my machine.

File: README.md (hunk 2 above)
Comment (sleipnir) [reply README.md:22]: Yeah! That was missing to mention, but we need it if we want to publish

**Comment by sleipnir** [comment]
> @sleipnir please find two comments above, after I tried it locally. I have nearly nothing python wise on my box, and this might be the reason the `wheel` module is not available on my machine.

I made some changes, see if it's cool

**Comment by marcellanz** [comment]
> > @sleipnir please find two comments above, after I tried it locally. I have nearly nothing python wise on my box, and this might be the reason the `wheel` module is not available on my machine.
> 
> I made some changes, see if it's cool

thank you @sleipnir, I can have a look tomorrow - today was a holyday and too much sun to enjoy 😀

**Comment by sleipnir** [comment]
@marcellanz  Very good, enjoy

**Comment by marcellanz** [comment]
Thank you @sleipnir

**Comment by marcellanz** [comment]
@sleipnir oops, closed the PR yesterday without merging. fixed!

**Pull request merged**